src/AppBundle/Controller/SemesterController.php

```diff
@@ -12,23 +12,19 @@
 
 class SemesterController extends Controller
 {
-    public function updateSemesterAction(Request $request)
+    public function updateSemesterAction(Request $request, Semester $semester)
     {
```

Comment:
Fordi her har vi en form. For å fange opp at formen har blitt submitted trenger vi `$request`: `$form->handleRequest($request);`